src/rules.ts

```diff
@@ -16,6 +16,16 @@ function edit(regex: string | RegExp, opt = '') {
   return obj;
 }
 
+const supportsLookbehind = (() => {
+try {
+  // eslint-disable-next-line prefer-regex-literals
+  return !!new RegExp('(?<=1)(?<!1)');
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+} catch(error) {
```

Comment:
```suggestion
} catch {
```

You should be able to remove the catch argument if it is not used

Reply:
Good catch! 😉